Documentation/CodingStyle.md

```diff
@@ -2,7 +2,12 @@
 
 For low-level styling (spaces, parentheses, brace placement, etc), all code should follow the format specified in `.clang-format` in the project root.
 
-**Important: Make sure you use `clang-format` version 19!**
+**Important: Make sure you use the correct version of `clang-format`!**
+
+See [lint-clang-format.sh](../Meta/lint-clang-format.sh) for the version enforced in CI.
```

Comment:
If we're going to refer to that script, can it be more readable? The actual version definition is halfway down the file, with no comments or anthing.

Reply:
I moved the version definition to the top of the file.

That also made me realize I forgot to commit changes to lint-clang-format.sh atomically, so now there's a commit that extracts the version to a single variable, and the switch-to-clang-format-20 commit just updates that variable.